### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_ScreenReader: NVDA is announcing the information "Discover the best customizations to make VS Code yours" twice.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#160410]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Aug2022; #WCAG1.3.1; #Info and Relationships; #Element:Button; #NVDA; #ScreenReader

### Environment and OS details:
Visual Studio Code
Windows 11 Pro version 21H2 (OS Build 22000.856)

### Repro Steps:
1)Open Visual studio code editor
2)Using "Alt+H" key navigate to the "Help" menu bar option.
3)Using "Down" arrow key navigate to the "Welcome" item.
4) Make sure you have the following setting in settings.json: "workbench.startupEditor": "gettingStarted"
5)Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started" and observe that the NVDA is announcing the information "Discover the best customizations to make VS Code yours" twice or not.

### Actual Result:
NVDA is announcing the information "Discover the best customizations to make VS Code yours" twice.

### Expected Result:
NVDA should not announce the "Discover the best customizations to make VS Code yours" twice it should be announced only once.

![160410_A11y_VisualStudioCodeClient_Welcome Screen_ScreenReader](https://user-images.githubusercontent.com/98530069/189136234-50bca047-b0ce-446f-9ede-38a76e5eba47.png)

**Comment by bradisbell** [comment]
![image](https://user-images.githubusercontent.com/1097509/204938817-4cb4089f-df65-4f79-96be-e1eba43d52e9.png)

Worse yet... it's just shoving this popup window on top of everything on my system now and won't go away.  Look, here's a screenshot of it floating on top of my browser window on this GitHub page.

I wish these ads would just go away.  The biggest problem with VSCode is that it's constantly trying to "enhance" my experience.  Reality is, every update breaks more things, resets my settings to some nonsense, and has more bugs than features.

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we are unable to reproduce the problem with the steps you describe. Chances are we've already fixed your problem in a recent version of VS Code. If not, please ask us to reopen the issue and provide us with more detail. Our [issue reporting guidelines](https://aka.ms/vscodeissuereporting) might help you with that.

Happy Coding!
